layouts/partials/download-list.hbs

```diff
@@ -5,7 +5,6 @@
     <li><a href="/{{site.locale}}/{{site.download.package-manager.link}}/">{{site.download.package-manager.text}}</a></li>
     <li><a href="/{{site.locale}}/{{site.download.releases.link}}/">{{site.download.releases.text}}</a></li>
     <li><a href="https://nodejs.org/download/nightly/">{{site.nightly}}</a></li>
-    <li><a href="https://nodejs.org/download/chakracore-nightly/">{{site.chakracore-nightly}}</a></li>
```

Comment:
Should the strings be cleaned out of the site.json too?

Reply:
> Should the strings be cleaned out of the site.json too?

Yes, indeed. Done!